inspect4py/cli.py

```diff
@@ -1249,7 +1249,7 @@ def main(input_path, fig, output_dir, ignore_dir_pattern, ignore_file_pattern, r
                 if ".py" in f and not f.endswith(".pyc"):
                     try:
                         path = os.path.join(subdir, f)
-                        out_dir = output_dir + "/" + os.path.basename(subdir)
+                        out_dir = output_dir + "/" + os.path.basename(subdir) # TODO: This could create path like /parent//child
```

Comment:
If you add a TO DO, please open an issue and point to this comment

Reply:
The issue has been created: https://github.com/SoftwareUnderstanding/inspect4py/issues/361#issue-1353004257.